ga4gh/backend.py

```diff
@@ -141,6 +161,37 @@ def startProfile(self):
     def endProfile(self):
         pass
 
+    def callSetsGenerator(self, request):
+        """
+        Returns a generator over the (callSet, nextPageToken) pairs defined by
+        the specified request.
+        """
+        # if no variantSetIds are input from client,
+        # then set variantSetIds to all variantSetIds
+        if request.variantSetIds == []:
+            variantSetIds = self._variantSetIds
+        else:
+            variantSetIds = request.variantSetIds
+        name = request.name
+        if request.pageToken is not None:
+            startVariantSetIndex, startCallSetPosition = self.parsePageToken(
+                request.pageToken, 2)
+        else:
+            startVariantSetIndex = 0
+            startCallSetPosition = 0
+
+        for variantSetIndex in range(startVariantSetIndex, len(variantSetIds)):
+            variantSetId = variantSetIds[variantSetIndex]
+            if variantSetId in self._variantSetIdMap:
+                variantSet = self._variantSetIdMap[variantSetId]
+                callSetIterator = variantSet.getCallSets(name,
```

Comment:
just as a matter of style I think this is easier to read like this (and easier to maintain if/when extra arguments are to be added):

```
                callSetIterator = variantSet.getCallSets(
                    name, startCallSetPosition)
```